modules/oe_content_timeline_field/src/Plugin/Field/FieldWidget/TimelineFieldWidget.php

```diff
@@ -27,9 +26,6 @@ class TimelineFieldWidget extends WidgetBase implements WidgetInterface {
    * {@inheritdoc}
    */
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
-    $element += [
```

Comment:
Why arent you validating anymore?

Reply:
That was for a specific field element. Since we don't need that anymore because if any of the fields are filled in is valid.